content/departments/engineering/teams/search/index.md

```diff
@@ -1,7 +1,112 @@
-# Search teams
+# Search Platform team
```

Comment:
Do we want to change the path to search-platform to avoid confusion with the code search team? (This can also happen in another PR)

Reply:
Done